website/assets/js/pages/handbook/basic-handbook.page.js

```diff
@@ -3,23 +3,64 @@ parasails.registerPage('basic-handbook', {
   //  ║║║║║ ║ ║╠═╣║    ╚═╗ ║ ╠═╣ ║ ║╣
   //  ╩╝╚╝╩ ╩ ╩╩ ╩╩═╝  ╚═╝ ╩ ╩ ╩ ╩ ╚═╝
   data: {
-    //…
+    isHandbookLandingPage: false,
+    showHandbookNav: false,
+    breadcrumbs: [],
+    subtopics: [],
+
   },
 
   //  ╦  ╦╔═╗╔═╗╔═╗╦ ╦╔═╗╦  ╔═╗
   //  ║  ║╠╣ ║╣ ║  ╚╦╝║  ║  ║╣
   //  ╩═╝╩╚  ╚═╝╚═╝ ╩ ╚═╝╩═╝╚═╝
   beforeMount: function() {
-    //…
+    if (this.thisPage.url === '/handbook') {
+      this.isHandbookLandingPage = true;
+    }
+    this.breadcrumbs = _.trim(this.thisPage.url, /\//).split(/\//);
+
   },
+
   mounted: async function() {
-    //…
+
+    this.subtopics = (() => {
+      let subtopics;
+      if(!this.isHandbookLandingPage){
+        subtopics = $('#body-content').find('h2').map((_, el) => el.innerText);
+      } else {
+        subtopics = $('#body-content').find('h3').map((_, el) => el.innerText);
+      }
+      subtopics = $.makeArray(subtopics).map((title) => {
+        // Removing all apostrophes from the title to keep  _.kebabCase() from turning words like 'user’s' into 'user-s'
+        let kebabCaseFriendlyTitle = title.replace(/[\’]/g, '');
+        return {
+          title,
+          url: '#' + _.kebabCase(kebabCaseFriendlyTitle),
+        };
+      });
+      return subtopics;
+    })();
   },
 
   //  ╦╔╗╔╔╦╗╔═╗╦═╗╔═╗╔═╗╔╦╗╦╔═╗╔╗╔╔═╗
   //  ║║║║ ║ ║╣ ╠╦╝╠═╣║   ║ ║║ ║║║║╚═╗
   //  ╩╝╚╝ ╩ ╚═╝╩╚═╩ ╩╚═╝ ╩ ╩╚═╝╝╚╝╚═╝
   methods: {
-    //…
+    isCurrentSection: function (section, location) {
```

Comment:
since this isn't directly invoked by a DOM event, please prefix the method name with `_` (e.g. `_isCurrentSection`) and update the HTML to match.